JavaScript/Topics/Async_Await/README.md

```diff
@@ -0,0 +1,353 @@
+# Async and Await
+
+JavaScript Introduces keywords “async” and “await” as an addition to EcmaScript 2015.We are going to see about the 
+following concepts. 
+
+1. Working With Async Keyword
+2. Impact of Await Keyword
+
+## What is Async and Await?
+
+Before getting in details of Async and Await, We should have a good understanding of Promises in JavaScript. 
+
+Async/Await is the extension of promises which we get as a support in the language. So before going to understand what
+is Async/Await, We should, first understand Promises in Javascript.
+
+### What is Promises in JavaScript?
+
+Promises are used to handle asynchronous operations in JavaScript.
+Promises are the ideal choice for handling asynchronous operations in the simplest manner. They can handle multiple 
+asynchronous operations easily and provide better error handling than callbacks and events.
```

Comment:
next line issue,

Please remove these things from the whole doc if it contains, i am not marking this further

Reply:
@Creatoon sir,
I prepared this documentation in notepad++. while reading the docs, there was a scroll at the bottom that's why I broke the line into several for convenience but forgot to remove those extra lines while making PR. My Fault. 
Thank you for the review. I will correct it.